### DISCUSSION TITLE
Adding a blog link to your podcast

### DISCUSSION THREAD

**Discussion by samsethi** [Podcastindex-org/podcast-namespace#648] [Tag Proposal]
Last week TrueFans proposed a new [events and ticketing](https://github.com/Podcastindex-org/podcast-namespace/discussions/640) solution to add to RSS feeds. i.e <podcast:events> 

### Example - Podcast Event Feed
```
<rss xmlns:podcast="https://podcastindex.org/namespace/1.0"
  xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd"
  xmlns:content="http://purl.org/rss/1.0/modules/content/"
  xmlns:atom="http://www.w3.org/2005/Atom" version="2.0">
<channel>
  <title>Nourish Your Network</title>
  <podcast:medium>podcast</podcast:medium>
  <podcast:events>
    <podcast:remoteItem medium="events" title="Nourish Your Network: The Confidence Booster Session" feedGuid="..." feedUrl="..."/>
  </podcast:events>
</channel>
</rss>
```

This week TrueFans wants to propose adding another new link to add your Blog feed. 

### Example - Podcast Blog Feed
```
<rss xmlns:podcast="https://podcastindex.org/namespace/1.0"
  xmlns:itunes="http://www.itunes.com/dtds/podcast-1.0.dtd"
  xmlns:content="http://purl.org/rss/1.0/modules/content/"
  xmlns:atom="http://www.w3.org/2005/Atom" version="2.0">
<channel>
  <title>Podnews Weekly Review</title>
  <podcast:medium>podcast</podcast:medium>
  <podcast:blog>
    <podcast:remoteItem medium="blog" title="Podnews Weekly Review: Blog" feedUrl="..."/>
  </podcast:blog>
</channel>
</rss>
```

### Image - TrueFans Admin Dashboard 

In TrueFans we have already created an UI in the podcasters Admin Dashboard.  This allows podcasters to create a blog post directly including images and/or video. 
 
<img width="1354" alt="Notification_Center" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/c9b0f867-fd2b-4ec6-88ac-a6ff9b451d9d">

### Image 2 - TrueFans Blog Post in App. 

The blog post will appear in the podcasters main page as a new tab.  However if the podcaster has an RSS feed they will be able to import their blog feed or Truefans will look for this new tag <podcast:blog> to find the Blog RSS feed URL.  

<img width="880" alt="Notification_Center" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/d23cc892-6278-4c9f-9be3-75c11f7ac7e0">

### Image 3 - TrueFans Blog Post with Boosts and Comments  

Readers can boost a post or leave a comment.

<img width="809" alt="Notification_Center" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/e8470dd9-0bfb-43d0-a0d4-db0781277731">

Version 2 will allow a podcaster to charge users to pay for the post before they can read it.

**Comment by jamescridland** [comment]
How do feedGuids work in this instance? Who sets them and where is a lookup for them?

**Reply by samsethi** [reply]
You're right James no need for a blog GUID. just list the RSS feed link.

**Comment by picard102** [comment]
Not sure I see the purpose of podcast apps also importing blog posts, or what the utility is to listeners.

**Reply by samsethi** [reply]
There are two types of apps now.  Single function per podcast apps like Overcast but many apps are becoming multi-function Spotify (music, podcasts, courses, video).  TrueFans wants to sit in the multi-function app side of the fence.  I want to show on a podcast or artist page live or IRL events, their blog, their podroll, what shows they have been a guest on and what podcasts their guests might host. We are also adding a link to a merch store.  We want the podcasters page to be an aggregated view.  We already offer blogs, podrolls and events. It's not for everyone but I do want to see if we can enable it in the RSS feed.

**Comment by nathangathright** [comment]
The existing spec already supports [`medium="blog"`](https://github.com/Podcastindex-org/podcast-namespace/blob/main/docs/1.0.md#medium) feeds, which can be linked from a publisher feed. If I understand correctly, I believe TrueFans could offer all you’ve shown here by taking advantage of the existing tags.

(This is the opposite of my position on events because those felt like “children” of individual podcasts, whereas blogs feel more like “siblings” to podcasts. For instance, there may be one blog for a network of podcasts rather than each show from a publisher having its own blog.)

**Reply by samsethi** [reply]
Again you are right, everything already exists.  I think I was not advocating for a new medium but a new podcast:blog as we have podcast:podroll and podcast:publisher - they are just markers in the RSS to contain either remote items or links. 
It's more about RSS/XML readability than functionality. 

We have now added the option to manually import your RSS feed but would love to read a podcast feed to grab the RSS blog feed.  

### EXAMPLE - An imported blog feed inside the podcasters  admin dashboard. 
<img width="1123" alt="Notification_Center" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/6ac623d3-5788-4457-9e92-ccdd3012340b">


### EXAMPLE - A blog feed as a tab inside the podcast page. 
For some reason this blog feed has no images.  We will refresh this blog every time the original blog feed is updated. 

<img width="1309" alt="Notification_Center" src="https://github.com/Podcastindex-org/podcast-namespace/assets/3492066/a577853b-cc51-46f4-aa26-0e62da275577">
